service/options.go

```diff
@@ -47,6 +48,17 @@ func WithMetrics(scope tally.Scope, reporter tally.CachedStatsReporter) Option {
 	}
 }
 
+// WithRootScope registers the root scope for all the manager metrics
+func WithRootScope(scopeFunc metrics.ScopeFunc) Option {
```

Comment:
Do we need this? Whats the benefit of having RootScope available as an option on service?